### PR TITLE
feat: allow form themes to have custom receipt template

### LINKED ISSUE #4545: feat: allow form themes to have custom receipt template
# Feature Request

## Developer Story

As a developer, I want to add support for the custom receipt template for form themes so that other developers can use it to create a custom receipt template.

## Acceptance Criteria

- [x] Core loads default receipt template if form theme does not load/define custom receipt template.
- [x] Developer can define a custom receipt template for form theme.

### PR DESCRIPTION
**Pull request by ravinderk**
## Description
This Pr will resolve #4545

## How Has This Been Tested?
This code test manually by editing the value of `Theme::view` in the form theme class.

## Checklist:
- [x] My code is tested.
- [x] My code follows the WordPress code style.
- [x] My code follows has proper inline documentation.

### CODE REVIEW

File: src/Form/Theme.php [hunk 1]
```diff
@@ -19,6 +19,17 @@
  * @since 2.7.0
  */
 abstract class Theme {
+	/**
+	 * view vs class array
+	 *
+	 * @since 2.7.0
+	 * @var array
+	 */
+	public $view = [
```
Comment (JasonTheAdams) [inline src/Form/Theme.php]: Please pluralize this: `views`

File: src/Form/Theme.php [hunk 2]
```diff
@@ -56,6 +67,20 @@ abstract class Theme {
...
+	/**
+	 * Theme template manager get template according to view.
+	 * Note: Do not forget to call this function before close bracket in overridden getTemplate method
+	 *
+	 * @param string $template
+	 *
+	 * @return string
+	 * @since 2.7.0
+	 */
+	public function getTemplate( $template ) {
```
Comment (JasonTheAdams) [inline src/Form/Theme.php]: We should be consistent here. Either change `$this->views` to `$this->templates`, or change `getTemplate` to `getView`.

Comment (ravinderk) [reply src/Form/Theme.php]: fixed: https://github.com/impress-org/givewp/compare/667bf05cc9e7...8584bd4f57e9

File: src/Form/Theme.php (hunk 1 above)
Comment (ravinderk) [reply src/Form/Theme.php]: fixed: https://github.com/impress-org/givewp/compare/667bf05cc9e7...8584bd4f57e9

**Pull request merged**